packages/caliper-cli/lib/bind/bind.js

```diff
@@ -15,6 +15,8 @@
 'use strict';
 
 const { CaliperUtils, ConfigUtil } = require('@hyperledger/caliper-core');
+
+const fs = require('fs-extra');
```

Comment:
The simple `fs` package also has an `existsSync` function, or is there a reason we don't like that?
https://nodejs.org/docs/latest-v8.x/api/fs.html#fs_fs_existssync_path